src/stream/src/executor/mview/materialize.rs

```diff
@@ -64,6 +64,10 @@ pub struct MaterializeExecutor<S: StateStore, SD: ValueRowSerde> {
     materialize_cache: MaterializeCache<SD>,
 
     conflict_behavior: ConflictBehavior,
+
+    metrics: Arc<StreamingMetrics>,
```

Comment:
We can use `streaming_metrics` in ` actor_context: ActorContextRef` instead of putting it in struct.

Reply:
thanks, let me update it